src/Coders/Console/CodeModelsCommand.php

```diff
@@ -86,7 +95,7 @@ protected function getConnection()
      */
     protected function getSchema($connection)
     {
-        return $this->option('schema') ?: $this->config->get("database.connections.$connection.database");
+        return $this->option('schema');
```

Comment:
Why did you change this line?

Reply:
because there is always something defined in the database config, it'll never be empty. The "mapAll" method that gets called when they don't force a specific schema will be called instead and handle whatever database they are connected to

Reply:
Would you mind making different PRs for each functionality you are trying to add? It'll be easier to review. I appreciate you're taking the time to contribute, btw. Thanks.

Reply:
I've noticed you are trying to add support for Postgress. That could be one PR. In addition, I see you are trying to refactor some code; that could be another PR. And please try not to change the code style, those changes make it really hard to spot the meaningful changes your are making to this package. I hope you won't mind.

Reply:
Well the postgres version can't run against the whole active database without that change :) that's why I had to add it

Reply:
yeah trying to keep it all to just postgres stuff, but PG is so different from MySQL/SQL that I needed more changes. Postgres has a tree separation of Schemas and Databases, as opposed to the other SQL's treating them as the same thing

Reply:
updated the style ci to use the laravel preset, hopefully that puts back all the style changes it made using the default styleci

Reply:
cool the style conflicts look resolved. but yeah without this config change to be directly params instead of combining the `database` concept with `schema`, the postgres version can't run. Cuz PG has a setup like `Server.[ Databases ].[ Schemas ].[ tables ]`
as opposed to the mysql style of `Server.[ Databases ].[ tables ]`

Reply:
so with postgres that means it'd try to run against a schema with the name of the databas

Reply:
The community will be very grateful for your contribution :)

Reply:
glad :) I have other ideas to add you might like... one being adding a type-hint phpdoc replacement for the Eloquent methods, so the IDE can know that things like `$object->where(...)` is going to return that model